twingate/manifest.json

```
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "c88bd253-18da-4224-af14-7854ce8ae6ed",
+  "app_id": "twingate",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Twingate provides a modern, Zero Trust alternative to corporate VPNs",
+    "title": "Twingate",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Twingate Activity Log",
+        "image_url": "images/twingate_activity_log.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Twingate Real-Time Activity Dashboard",
+        "image_url": "images/dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Category::Network",
+      "Category::Security",
+      "Category::Cloud",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Google Cloud"
+    ]
+  },
+  "author": {
+    "support_email": "support@twingate.com",
+    "homepage": "https://www.twingate.com/?utm_source=datadog&utm_medium=partner&utm_campaign=integrations",
+    "sales_email": "sales@twingate.com",
+    "name": "Twingate"
+  },
+  "oauth": {},
+  "assets": {
+    },
+    "dashboards": {
+      "Twingate Dashboard": "assets/dashboards/twingate_overview.json"
+    }
```

```suggestion
 "assets": {
    "dashboards": {
      "Twingate Dashboard": "assets/dashboards/twingate_overiew.json"
    }
  }
```

`dashbords` should be in the `assets` section